src/styles/global.css

```
@@ -3592,21 +3592,233 @@ body.resume-page {
+.resume-canvas-topics {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.3rem;
+}
+.resume-canvas-topic {
+  display: inline-flex;
+  align-items: center;
+  padding: 0.18rem 0.45rem;
+  background: rgba(17, 16, 13, 0.08);
+  border-radius: 2px;
+  font-family: "Inter", system-ui, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)